### DISCUSSION TITLE
Quick-Save buttons

### DISCUSSION THREAD

**Discussion by Sergorov** [Northfear/fallout2-ce-vita#7] [Ideas]
I experienced a few crashes by now, and so I was wondering: Is there a way to map f6 (quick-save) and f7 (quick-load) to the shoulder buttons? As saving requires quite a lot of button pushes right now it's becoming a nuisance. But given that this is under development, and more or less frequent crashes are to be expected, it would help having the quick-save feature easily accessible I think.

**Comment by Northfear** [comment]
I think shoulder button combo like select+l/r would work better (to avoid accidental presses). Save slots can be corrupted tho, which could be a problem with over-reliance on quicksaves. I'll look into it.

**Reply by Sergorov** [reply]
Good idea re the combination! Yes, I thought about corrupted savegames too. Still I think it is worth doing it. Everyone who can read should be aware, additionally you could add a remark to the readme (where you would add the shortcut, if you do ;-)). I would still save my game "properly" when completing a quest or entering a new town. And I would always use a separate slot just for quick-saves. Trust your users, we're not that bad!

**Comment by Northfear** [comment]
Should be added in the latest development build
https://github.com/Northfear/fallout2-ce-vita/releases/tag/latest

**Reply by Sergorov** [reply]
Did I mention you're absolutely amazing? Tested a bit and it works like a charm! 
I also love how the screen is used beyond the main UI bar in the latest build. 
Well, have I mentioned that you are absolutely, freaking AMAZING??

**Reply by Northfear** [reply]
Heh, thanks. All of the credits should go to the original author tho. I've never thought that I would see Fallout 2 on Vita in my lifetime

**Reply by Sergorov** [reply]
Not all. You're the Vita guys, and it's fabulous (and a little crazy) that it runs on my favourite abandoned (yet adopted and risen by a loving community) handheld console. Thank you for that.
